lib/twingly/url.rb

```diff
@@ -6,7 +6,12 @@
 require_relative "url/error"
 require_relative "version"
 
-PublicSuffix::List.private_domains = false
+PublicSuffix::List.default = PublicSuffix::List.parse(
+  File.read(PublicSuffix::List::DEFAULT_LIST_PATH), private_domains: false)
+PublicSuffix::List.default.indexes.keys.
+  map { |name| Addressable::IDNA.to_ascii(name) }.
+  select { |name| name =~ /xn\-\-/ }.
```

Comment:
I think `/\Axn\-\-/i` is the correct regex.

> The ToASCII and ToUnicode operations MUST recognize the ACE prefix in a case-insensitive manner.

> The ACE prefix for IDNA is "xn--" or any capitalization thereof.

https://tools.ietf.org/html/rfc3490#section-5